### ISSUE TITLE
Implement custom headers

### ISSUE THREAD

**Issue by Lex0ne** [swindon-rs/swindon#59]
Need to know, from which frontend received request by websocket call. I think, it would be great, if we have opportunity to mark routes in swindon config by label and then get label from header in each request.

**Comment by tailhook** [comment]
Any reason why [extra-headers](https://swindon-rs.github.io/swindon/config/handlers.html#opt-extra-headers) doesn't work?

**Comment by popravich** [comment]
`extra-headers` only work for static files.
I guess we can make this setting common to all handlers.
Your thoughts?

**Comment by tailhook** [comment]
Yes. Just be careful not to replace content-length or transfer-encoding headers.
